src/sentry/conf/server.py

```diff
@@ -787,11 +782,6 @@ def create_partitioned_queues(name):
         "schedule": timedelta(minutes=20),
         "options": {"expires": 20 * 60},
     },
-    "check-symbolicator-lpq-project-eligibility": {
-        "task": "sentry.tasks.low_priority_symbolication.scan_for_suspect_projects",
-        "schedule": timedelta(seconds=10),
-        "options": {"expires": 10},
-    },
```

Comment:
i'm open to keeping all of these changes around. however, this means that this task which runs incredibly frequently does literally nothing in the background by default. i think a case can be made for moving this out into `getsentry`, and to give users instructions on how to enable the LPQ in a non-SaaS setting should they really need it.

Reply:
using a different approach. i've set this to run on feb 32 in sentry so it never runs. 32 so if feb magically gets the same number of days as other months, it's still out of range for a "regular" month.

Reply:
*31 days because some amount of validation is run on these values so 32 is not possible, but feb 31 is a-okay